src/triton_cli/parser.py

```diff
@@ -502,5 +437,76 @@ def parse_args(argv=None):
     parse_args_profile(subcommands)
     parse_args_utils(subcommands)
     add_verbose_args([parser])
-    args = parser.parse_args(argv)
+
+    argv_ = argv if argv is not None else sys.argv[1:]
+    # If a passthrough command is passed as the first arg,
+    # special handling is needed.
+    if argv_[0] == "profile":
+        args, unknown_args = parser.parse_known_args(argv_)
+        args = add_unknown_args_to_args(args, unknown_args)
+    else:
+        args = parser.parse_args(argv_)
     return args
+
+
+# ================================================
+# Helper functions
```

Comment:
Can you move all these helpers to `profile.py` and then:
```python
from profile import build_command
...
def handle_profile(args: argparse.Namespace):
    cmd = profile.build_command(args, executable="genai-perf")
    logger.info(f"Running: '{' '.join(cmd)}'")
    subprocess.run(cmd, check=True)
```

Reply:
Sure, done!